mercadopago/px-ios
# (Added)[PXN-3140] - Added paymentMethodRuleSet

## What have changed?
We've added paymentMethodRuleSet to init checkout request

## How to test:
You can add this line to paymentMethodSelector builder func on ExampleApp:

builder.setPaymentMehtodRuleSet(paymentMethodRuleSet: ["IGNORE_INSUFFICIENT_AM_BALANCE"])

And check that the field is being send.
